src/legal-officer/RecoveryDetails.tsx

```diff
@@ -54,26 +54,44 @@ export default function RecoveryDetails() {
         }
     }, [ axiosFactory, accounts, recoveryInfo, setRecoveryInfo, requestId ]);
 
+    const alreadyVouched = useCallback(async (lost: string, rescuer: string, currentAddress: string) => {
+        const activeRecovery = await getActiveRecovery({
+            api: api!,
+            sourceAccount: lost,
+            destinationAccount: rescuer
+        });
+
+        return !!(activeRecovery && activeRecovery.legalOfficers.find(lo => lo === currentAddress));
```

Comment:
Just for my information, what does `!!` exactly mean?

Reply:
It's called "notnot" - to covert any type to boolean. In this case it was a suggestion of my IDE, not sure it was necessary.